src/styles/blocks/card.scss

```diff
@@ -0,0 +1,125 @@
+@import "../partials/variables.scss";
+
+.card {
+  max-width: 198px;
+  min-height: 406px;
+  transition: all 300ms;
```

Comment:
the transition should be more specific, please target only CSS properties you will be animating